binding/java/src/ai/picovoice/porcupine/Utils.java

```diff
@@ -34,6 +34,8 @@
     private static final String ENVIRONMENT_NAME;
     private final static Logger logger = Logger.getLogger(Logger.GLOBAL_LOGGER_NAME);
 
+    private static String linuxArch;
```

Comment:
To keep it more flexible and clean, why don't we have a static function in the static block below that gets OS and Arch and then we'll use those in the rest of the functions.